### PR TITLE
Fix compilation for crystal 0.27.2

### LINKED ISSUE #12: Build failure on head
There's too many issues to list, so it needs some work with someone more familiar with the codebase.

```
Crystal 0.27.2 (2019-02-05)

LLVM: 6.0.1
Default target: x86_64-apple-macosx
```

### PR DESCRIPTION
**Pull request by steakknife**
Closes #12

### CODE REVIEW

File: h.cr [hunk 1]
```diff
@@ -27,15 +27,17 @@ server = HTTP::Server.new(host, port) do |context|
   end
 end
 
-if ENV["CI"]?
-  server.logger = Logger::Dummy.new(File.open("/dev/null"))
-end
+#if ENV["CI"]?
```
Comment (Sija) [inline h.cr]: What's the situation here? Should this be uncommented, removed, ...?

Comment (ysbaddaden) [reply h.cr]: This should be left in.

Comment (steakknife) [reply h.cr]: The `logger` property no longer exists or moved, so it failed to compile. Where did it move to?

**Comment by steakknife** [comment]
`h.cr` is broken (doesn't compile), however it's unclear which server interface is correct.

**Comment by ysbaddaden** [comment]
If I recall correctly, the current test server is `/server.cr` which wraps `src/server.cr`. The `/h.cr` and `/h2.cr` are legacy HTTP/2 only servers.

Note that if you're interested, I pushed an [old branch](https://github.com/ysbaddaden/http2/tree/http-takeover) that was trying to take over the HTTP namespace from stdlib, to transparently, but that was last updated some years ago, and there are a bunch of broken things.

**Pull request merged**